Fix: If filter hits on MyPlan page, filter button is shown

t-kashima opened the pull request:
## Issue
- close #397

## Overview (Required)

- I changed the conditions for displaying the filter button.
- The filter button is displayed when (has filtered sessions || unfiltered) && close the filter.

## Links
-

## Screenshot
Before | After
:--: | :--:
<img src="https://user-images.githubusercontent.com/966751/72666407-43a8f980-3a55-11ea-9441-74657946971d.png" width="300"> | <img src="https://user-images.githubusercontent.com/966751/72666408-499eda80-3a55-11ea-8c7c-77c27964fa21.png" width="300">

takahirom commented:
Thanks!
Probably when there are no favorite sessions, the image should be displayed, so the button should not be displayed. 🖼

t-kashima commented:
Thank you for reviewing!
I changed to not show the filter button when there is no session. 
And I found the filter button is not displayed when there are sessions but no filtered sessions and fixed it.

Please review again 🙏

takahirom commented on feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetFavoriteSessionsFragment.kt:
```diff
@@ -80,19 +80,7 @@ class BottomSheetFavoriteSessionsFragment : DaggerFragment() {
-                    View.INVISIBLE
-            }
-            binding.expandLess.isVisible = when (uiModel.expandFilterState) {
+            binding.isCollapsed = when (uiModel.expandFilterState) {
```
👏

t-kashima commented:
Thank you for the review😊
